### PR TITLE
Refactoring inma

### PR DESCRIPTION
**Pull request by inmasalcedo**
Can you check my refactoring? It's from line 303 till 507. It has a huge function "inputs" that I tried to make it smaller or easier to read but I couldn't do it, maybe you have more ideas.

### CODE REVIEW

File: old_version/assets/js/main.js [hunk 1]
```diff
@@ -296,15 +298,17 @@ inputRed.addEventListener("click", handleColorTheme);
...
 const ubuntuFont = document.getElementById("font-ubuntu");
 const comicFont = document.getElementById("font-comic");
 const montseFont = document.getElementById("font-montse");
 const fontCard = document.querySelector(".card-header");
 
-function handleFonttheme() {
+function handleFonttheme(event) {
   fontCard.classList.remove("font-ubuntu", "font-comic", "font-montse");
 
   const fontSelectedByUser = event.currentTarget;
```
Comment (beatrizAdalab) [inline old_version/assets/js/main.js:314]: maybe we can use ternary here:

```
fontSelectedByUser === ubuntuFont ? fontCard.classList.add("font-ubuntu"):
fontSelectedByUser === comicFont ? fontCard.classList.add("font-comic"):
fontSelectedByUser === montseFont ? fontCard.classList.add("font-montse"):console.log("undefined");
```
**Attention!**When refreshing typography is not updated, although if saved in localStorage. Does the same thing happen to you? Maybe we need to review it.

File: old_version/assets/js/main.js [hunk 2]
```diff
@@ -397,6 +401,17 @@ let skillArray = [];
 let j = 0;
 let acc = 0;
 
+function gettingSkills() {
```
Comment (beatrizAdalab) [inline old_version/assets/js/main.js:404]: Maybe we could use ternaries again?

```
skillArray = dataCard.skills,
        j = skillArray.length,
        acc = skillArray.length
      ): skillArray = [];
```

**Review by beatrizAdalab — CHANGES_REQUESTED** [review]
Hello, I'm sending you some ideas to reduce conditional the rest I see it well. The big function, I can not think of any improvement at the moment.

File: old_version/assets/js/main.js (hunk 2 above)
Comment (inmasalcedo) [reply old_version/assets/js/main.js:404]: I tried before but I'm not sure about the ternary syntax in this case. 

(dataCard.skills) ?
    skillArray = dataCard.skills;
    j = skillArray.length;
    acc = skillArray.length;
  :
    skillArray = [];

EsLint asks me to change the first  ; to a  : and I tried to put in brackets the skill Array, j and acc but It doesn't work either...

**Comment by inmasalcedo** [comment]
This pull request is no longer necessary.

**Pull request closed without merging**